vad.py

```diff
@@ -53,14 +11,118 @@ def resize_to_min_size_(*tensors, dim = -1):
 			sliced = t.narrow(dim, 0, size)
 			t.set_(t.storage(), 0, sliced.size(), sliced.stride())
 
-class WebrtcSpeechActivityDetectionModel(nn.Module):
-	def __init__(self, aggressiveness):
+
+class PrimitiveVAD:
+	def __init__(self,
+	             device: str = 'cpu',
```

Comment:
Can we keep the device casts out of the model itself? even more so if the model is parameter-less